tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs

```diff
@@ -125,5 +123,24 @@ private static bool VerifyAlias(string uriStub)
                 throw;
             }
         }
+        private static IEnumerable<CodeOwnerEntry> findOwnerEntries(string target, List<CodeOwnerEntry> parsedEntries)
```

Comment:
```suggestion
        private static CodeOwnerEntry FindClosestMatch(string target, List<CodeOwnerEntry> parsedEntries)
```

This function should also be added to the CodeOwners library not here.

Reply:
Thinking about this we should have it be a helper function similar to:

https://github.com/Azure/azure-sdk-tools/pull/2085/files#diff-143b558cd0d0e785a6dd1718176854379e72bcbd3acd114f16fd3cb68924ec2eL37
```
        public List<string> GetContactsForPath(string filePath)
        {
            // Normalize the start and end of the paths by trimming slash
            filePath = filePath.Trim('/');

            // We want to find the match closest to the bottom of the codeowners file
            for (int i = expressionContacts.Count - 1; i >= 0; i--)
            {
                string pathExpression = expressionContacts[i].GlobExpression.Trim('/');

                // Note that this only matches on paths without glob patterns which is good enough
                // for our current scenarios but in the future might need to support globs
                if (filePath.StartsWith(pathExpression, StringComparison.OrdinalIgnoreCase))
                {
                    return expressionContacts[i].Contacts;
                }
            }

            return new List<string>();
        }
```

It should take in the path to the codeowners files and the target path and return a list of github alias' for it.

Reply:
Will take a look. Thanks for helping direct me to the right place. :)